src/improvements/tasks/MultisigTask.sol

```diff
@@ -30,17 +30,13 @@ abstract contract MultisigTask is Test, Script, StateOverrideManager, TaskManage
     using AccountAccessParser for VmSafe.AccountAccess;
     using StdStyle for string;
 
-    /// @notice Parent nonce used for generating the safe transaction.
-    uint256 public nonce;
+    /// @notice The root safe address for the task
+    address public root;
```

Comment:
will users (i.e. task developers) ever need to use this variable? A more clear name might be useful
```suggestion
    address public rootSafe;
```

Reply:
Maybe just leave a `// TODO:` so that it is not completely forgotten